scripts/processAndDownloadNFTs.ts

```diff
@@ -97,7 +97,7 @@ async function fetchOpenseaFromUsers(nftOwners: string[]) {
 					featured: collection.featured,
 					protocol: asset.schema_name || undefined,
 					symbol: asset.symbol,
-					logoURI: logoURI,
+					logoURI: logoURI || '/images/nfts/unknown.png',
```

Comment:
Same as above, I think `undefined` is the correct choice here.

Reply:
Ugh, i know, but the way the json is read in, it doesn't understand that undefined only appears in the Nth field of the json

Reply:
I wonder if we can integrate something like this into our build system to make get better, but still automatic, type checking?
https://github.com/microsoft/TypeScript/issues/32063#issuecomment-800162987

Reply:
Another script further down the thread: https://github.com/microsoft/TypeScript/issues/32063#issuecomment-916071942

Reply:
> That seems way more cumbersome than just checking each value against undefined

While I agree it is cumbersome, the critical part is that it is cumbersome in this project rather than in *every project that uses this as a dependency*.  We should have strong type checking where possible, such as on the name and symbol.  We shouldn't be just throwing our hands up in the air because one item in the tuple could be undefined.

If we don't care about data size, we could switch to an object instead of a tuple, which would result in the inferred type being correct.  After compression (zipping) it likely will come out similarly sized, but uncompressed it'll end up being substantially bigger.